core/src/main/java/io/grpc/internal/AbstractClientStream.java

```diff
@@ -174,7 +174,7 @@ public final boolean isReady() {
     private ClientStreamListener listener;
 
     private boolean deframerClosed = false;
-    private Runnable deframerClosedTask;
+    protected Runnable deframerClosedTask;
```

Comment:
Yeah, that looks like a comment that failed to be sent out earlier. Sorry for the noise.